### ISSUE TITLE
Building debian packages fails to create a proper install package

### ISSUE THREAD

**Issue by draekko** [input-leap/input-leap#1602]
### What happened?

Building debian packages fails to create a proper install package.

No binary files, man pages or readme/doc are included when building the packages.

### Version

From Git HEAD or commit (specify below)

### Git commit hash (if applicable)

HEAD

### If applicable, where did you install Input Leap from?

git master compiled from source code

### What OSes are you seeing the problem on? (Check all that apply)

Linux

### What OS versions are you using?

Ubuntu 22.10

### Any other information

Including a patch here as i have another PR waiting approval to make it available in the meantime.

The patch was ported from the one i had for barrier.

https://gist.github.com/draekko/536a85dcaee1a1cf2a96a0f8853080ef

**Comment by draekko** [comment]
Just noticed that the desktop file for Linux still references barrier and will not launch the correct app. Additional patch below to resolve that.

https://gist.github.com/draekko/c89db62a66242613aecb6afe42a2cb0d

**Comment by SteveDevOps** [comment]
Noticed as well after installing via yay on Arch -- how to build with yer patch? (first time using one)

**Comment by draekko** [comment]
The first patch is for Debian based distros, the second can be applied irrespective of distro, for Arch i have no idea what else you'd need to do, for debian/ubuntu/etc do:

cd /path-to-repo
ln -sf  dist/debian .

Then do this if you use deb tools

git submodule update --init --recursive
dpkg-buildpackage -uc -b

Or this if you have git-buildpackage installed and prefer that route

gbp buildpackage --git-submodules -b -uc --git-export-dir="latest"  --git-no-create-orig --git-ignore-new

**Comment by SteveDevOps** [comment]
sorry pulled my Q as I realized you were referring to debian specifically
-- but appreciate the feedback. ;)

**Comment by shymega** [comment]
Please submit the PR, instead of a Gist. Makes it easier to track. Your other pending PR has been converted to a draft until there's time to push the changes I requested.

The two Gists should be separate PRs, but Debian package fixes are welcome :+1:

**Comment by shymega** [comment]
I've made a PR - #1613 - I don't use Debian, so I can't test the scripts yet, unless I spin up a container. 

@draekko I've made it a co-authored commit with you, and I've signed off on it. Just so you get the credit for the changes :-)

**Comment by draekko** [comment]
Sorry for the late reply been busy, thanks for handling that and the other PR.

**Comment by shymega** [comment]
@draekko Can you test the PR? Not familiar with Debian packaging, more focused on Nix/Flatpak/Snap, for Linux.

**Comment by draekko** [comment]
Just patched from that PR and it worked here.

**Comment by chr-chr** [comment]
Hi,

this build for me.
You may add to debian/control:

```
Build-Depends: cmake,
 libavahi-compat-libdnssd-dev,
 libcurl4-openssl-dev | libcurl-dev,
 libgtest-dev,
 libice-dev,
 libsm-dev,
 libssl-dev,
 libxi-dev,
 libxinerama-dev,
 libxrandr-dev,
 libxtst-dev,
 libgmock-dev,
 qtbase5-dev,
 qttools5-dev,
 xvfb,
 xauth
```

TODO: need a version number, 2.4.0(master source) vs. 3.0.0 (debian/changelog)

**Comment by shymega** [comment]
> Hi,
> 
> this build for me. You may add to debian/control:
> 
> ```
> Build-Depends: cmake,
>  libavahi-compat-libdnssd-dev,
>  libcurl4-openssl-dev | libcurl-dev,
>  libgtest-dev,
>  libice-dev,
>  libsm-dev,
>  libssl-dev,
>  libxi-dev,
>  libxinerama-dev,
>  libxrandr-dev,
>  libxtst-dev,
>  libgmock-dev,
>  qtbase5-dev,
>  qttools5-dev,
>  xvfb,
>  xauth
> ```
> 
> TODO: need a version number, 2.4.0(master source) vs. 3.0.0 (debian/changelog)

Please make a PR to add these changes once #1613 is merged.

**Comment by chr-chr** [comment]
This bureaucracy ... just copy and paste these lines into our PM :grin:

**Comment by shymega** [comment]
Fixed by #1613.
